scipy/integrate/__init__.py

```diff
@@ -30,11 +30,11 @@
 .. autosummary::
    :toctree: generated/
 
-   trapz         -- Use trapezoidal rule to compute integral.
-   cumtrapz      -- Use trapezoidal rule to cumulatively compute integral.
-   simpson       -- Use Simpson's rule to compute integral from samples.
-   romb          -- Use Romberg Integration to compute integral from
-                 -- (2**k + 1) evenly-spaced samples.
+   trapezoid            -- Use trapezoidal rule to compute integral.
+   cumulative_trapezoid -- Use trapezoidal rule to cumulatively compute integral.
```

Comment:
3 line-too-long warnings in integrate/ files
$ tools/lint_diff.py
439./scipy/integrate/__init__.py:34:80: E501 line too long (81 > 79 characters)
440./scipy/integrate/_quadrature.py:31:80: E501 line too long (80 > 79 characters)
441./scipy/integrate/tests/test__quad_vec.py:9:80: E501 line too long (80 > 79 characters)

Reply:
Yes, I chose to ignore that - breaking the lines for 1 or 2 characters doesn't make sense here. The CI failure won't show up in subsequent PRs, so ignoring is fine.

Reply:
I would put a `# noqa: E501` at the end of the triple-quoted docstring (line 89). That way editors won't complain, either.

Reply:
Makes sense, thanks - done.